internal/platform/gin/auditlog/auditlogdriver/model.go

```diff
@@ -1,4 +1,4 @@
-// Copyright © 2018 Banzai Cloud
+// Copyright © 2020 Banzai Cloud
```

Comment:
**Note**: changing copyright notices to current year can be done en masse with a simple script, I have some previous experiences with it, let me know if you would like to see a PR doing just that.

Reply:
This was done by the IDE automatically (I actually copied the content to a new file which includes the header with the new year)